appservice/src/createAppService/createWebApp.ts

```diff
@@ -23,17 +27,15 @@ export async function setWizardContextDefaults(wizardContext: IAppServiceWizardC
     await LocationListStep.setLocation(wizardContext, 'centralus');
```

Comment:
I'm beginning to think `setWizardContextDefaults` should be directly in the appservice repo rather than in the shared package. (And then you'd pass that stuff in as a part of `IAppCreateOptions`). Thoughts? Doesn't have to be done in this PR.

Reply:
Yeah, I agree.  I think it's starting to get a little bit messy and seems like there'll be some options for Functions that will just make it a lot of branching logic.